documentation/CHANGELOG_EXAMPLE.md

```diff
@@ -0,0 +1,474 @@
+- **This is only the pre-release, it should be considered as a draft of the upcoming release v0.10.0**
+  - The pre-release is meant for implementors, but it is not recommended to use the API with customers in productive environments.
+
+**Main Changes**
+
+* Aligned event notification with CloudEvent spec which will allow API consumers and implementators to use standard libraries and tools which are available to handle CloudEvents (https://cloudevents.io/)
+* Added a new operation `/sessions/{sessionId}/extend` which allows to extend the duration of an active session
+
+- API definition **with inline documentation**:
+  - [View it on ReDoc](https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/camaraproject/QualityOnDemand/v0.10.0-rc/code/API_definitions/qod-api.yaml&nocors)
+  - [View it on Swagger Editor](https://editor.swagger.io/?url=https://raw.githubusercontent.com/camaraproject/QualityOnDemand/v0.10.0-rc/code/API_definitions/qod-api.yaml)
+  - OpenAPI [YAML spec file](https://github.com/camaraproject/QualityOnDemand/blob/v0.10.0-rc/code/API_definitions/qod-api.yaml)
+
+## Changes for quality-on-demand v0.10.0-rc.1
+
+### Added
+
+* Added new endpoint to extend duration of an active session by @emil-cheung in https://github.com/camaraproject/QualityOnDemand/pull/216
+* Introduced of linting with Megalinter and Swagger Editor Validator by @RandyLevensalor, @maxl2287 and @ravindrapalaskar17 in https://github.com/camaraproject/QualityOnDemand/pull/206, https://github.com/camaraproject/QualityOnDemand/pull/207, https://github.com/camaraproject/QualityOnDemand/pull/212, and  https://github.com/camaraproject/QualityOnDemand/pull/215
+* Added global tags element  by @rartych in https://github.com/camaraproject/QualityOnDemand/pull/227
+
+### Changed
+
+* Align event notification with CloudEvents spec by @jlurien in https://github.com/camaraproject/QualityOnDemand/pull/224
+* Moved "description" out of "allOf" declaration by @maxl2287 in https://github.com/camaraproject/QualityOnDemand/pull/205
+  * Note: this change shouldn't have an impact for API consumers but is relevant for implementations of the API.
+* Aligned with changes in https://github.com/camaraproject/Template_Lead_Repository on test definitions by @rartych in https://github.com/camaraproject/QualityOnDemand/pull/233
+* Single IP addresses in Device model specified with standard formats instead of patterns by @jlurien in https://github.com/camaraproject/QualityOnDemand/pull/237
+
+### Fixed
+
+* NA
+  
+### Removed
+
+* NA
+
+## New Contributors
+* @ravindrapalaskar17 made their first contribution in https://github.com/camaraproject/QualityOnDemand/pull/215
+* @rartych made their first contribution in https://github.com/camaraproject/QualityOnDemand/pull/227
+
+**Full Changelog**: https://github.com/camaraproject/QualityOnDemand/compare/v0.9.0...v0.10.0-rc
+
+# r2.2 - alpha release
+
+## Release Notes
+
```

Comment:
From here downwards I propose to just mention that there is the original content

```suggestion
** NOTE: the following releases are done before the introduction of the Release Process: **

# v0.9.0

...


```

Reply:
Included the Note